### PR TITLE
Das G. Ma - testing

### PR DESCRIPTION
**Pull request by DasGMA**

### CODE REVIEW

File: basic-javascript/project-1.spec.js [hunk 1]
```diff
@@ -1,3 +1,236 @@
...
+
+
+it('checks for roundup number', () => {
+    const roundUp = helpers.roundUp;
+
+    const upRound = roundUp(1.95);
+
+    expect(upRound).toBe(2);
+});
+
```
Comment (kelfro) [inline basic-javascript/project-1.spec.js]: I think for the previous 5 tests, maybe you could make the names more specific. Because it's not really checking for a square, it's squaring the number(s) provided, or cubing them, or rounding them up or down.

**Comment by kelfro** [comment]
Great job achieving MVP and doing the advanced js part, too! Maybe today you can work on more edge cases - like, what happens if you enter a string where number or an array is supposed to go? Or an object where it should be an array? Or divide by 0? What happens if your test is looking for a string with lowercase but someone enters something with a capital letter? If necessary, you can edit the original project files that you are testing to help with sanitizing the data entry so that only the correct types (number, string, etc.) can be entered.
Nice work!

**Comment by kelfro** [comment]
Stretch goal completed - nice! All of your tests look well written and they all check out in the test suite. Would be nice to see more variety in the testing, including edge cases, but all of this is solid and works correctly. Good work!
